### PR TITLE
fix production env settings

### PR DESCRIPTION
**Pull request by royxue**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ module.exports = generators.NamedBase.extend({
     }, {
       type: 'input',
       name: 'userName',
-      message: 'What is your name? (Used for the package.json.)',
+      message: 'What is your name? (Used for the package.json. Capitalize the first character)',
```
Comment (derekpeterson) [inline app/index.js]: Will you undo this change? This isn't actually related to the component name.

File: app/templates/gulpfile.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ gulp.task('js:prod', function (done) {
   prodConfig.plugins = prodConfig.plugins.concat(
 		new webpack.DefinePlugin({
 			'process.env': {
-				NODE_ENV: 'production'
+				NODE_ENV: JSON.stringify("production")
```
Comment (derekpeterson) [inline app/templates/gulpfile.js:39]: Good catch. I removed that without even thinking about it.

File: app/index.js (hunk 1 above)
Comment (royxue) [inline app/index.js]: OK, sorry I made a mistake here I think

Comment (royxue) [inline app/index.js]: @derekpeterson But I think this should be added at the name of project part, what do you think

Comment (derekpeterson) [inline app/index.js]: @royxue That value is transformed and [used three ways](https://github.com/derekpeterson/generator-react-flask/blob/master/app/index.js#L57): kebab-case (package.json), snake_case (Python), and PascalCase (React component). Since we override the case that they specify, it's not a big deal.

**Comment by derekpeterson** [comment]
@royxue Sorry for the slow reply! I haven't been getting notifications for this repo. Everything looks good. Thank you for doing this!
